common/components/TXMetaDataPanel/components/SimpleGas.tsx

```diff
@@ -76,6 +76,16 @@ class SimpleGas extends React.Component<Props> {
       min: gasEstimates ? gasEstimates.safeLow : gasPriceDefaults.min
     };
 
+    /**
+     * @desc On retrieval of gas estimates,
+     *  the current gas price may be lower than the lowest recommended price.
+     *  `rc-slider` will force the onChange if the value is too low, so we
+     *  ensure it at least passes the lower boundary.
+     *  When this occurs, the logic in `UNSAFE_componentWillReceiveProps` fires,
+     *  and it cannot happen again from that point forward.
+     */
+    const actualGasPrice = Math.max(this.getGasPriceGwei(gasPrice.value), bounds.min);
```

Comment:
I believe this will work fine, because the time between the value still being the initial default and it being changed as a result of `componentWillReceiveProps` in `SimpleGas` is almost instant.